GOFevaluation/utils.py

```diff
@@ -103,14 +100,9 @@ def limit_bin_edges(bin_edges, data_sample, order):
     return be
 
 
-def density_modifier(ns, be_first, be_second, data_sample):
-    """Creates limits for the edges of bins. Normally, bins
-    at the borders are infinite, this allows for all possible
-    datapoints to be binned. However, this makes it impossible
-    to calculate the areas of these border bins, and to show the
-    edges of these bins in a plot. This function uses the limits
-    and data_sample to limit the total area of all of the bins
-    so that they can be plotted and their area can be measured.
+def get_count_density(ns, be_first, be_second, data_sample):
```

Comment:
I'm very sorry to ask for another change here.. But to make it consistent we should make this function private (i.e. `_get_count_density`)

Reply:
Same for `get_finite_bin_edges`